pkg/unikontainers/storage.go

```diff
@@ -15,45 +15,156 @@
 package unikontainers
 
 import (
+	"bufio"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
 	"strings"
 
-	"github.com/shirou/gopsutil/disk"
+	"github.com/moby/sys/mount"
+	"github.com/sirupsen/logrus"
 )
 
-// RootFs represents a root file system and its properties.
+var ErrMountpoint = errors.New("No FS is mounted in this mountpoint")
+
+// RootFs contains information regarding a mount
 type RootFs struct {
-	Path      string             // The path of the root file system.
-	IsBlock   bool               // Indicates if it's a block device.
-	BlkDevice disk.PartitionStat // Information about the block device.
+	Path   string // The path of the root file system.
+	Device string // The device which is mounted as the container rrotfs
```

Comment:
Just a minor comment, rrotfs should be rootfs.

Reply:
Yes, I will fix that.